packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderThemeStyleTag.ts

```diff
@@ -76,5 +86,22 @@ export const useFluentProviderThemeStyleTag = (options: Pick<FluentProviderState
     }
   }, [styleTagId, targetDocument, rule, styleElementAttributes]);
 
-  return styleTagId;
+  return { styleTagId, rule };
```

Comment:
Is it a breaking change?

Reply:
https://github.com/microsoft/fluentui/blob/b51bba0138091bf39add65e831d4b754f386d4cf/packages/react-components/react-provider/src/index.ts#L8

Reply:
why would it be? we don't export this hook

Reply:
do we consider it a part of our public API surface?

Reply:
I can slap `@internal` on this utility (IMO it should have been like that to begin with). does that address the concern here?